tests/extras/test_translator.py

```diff
@@ -96,18 +155,49 @@
 }
 
 
+duplication_output_normalized = {
+    "location": {
+        "end": 19993839,
+        "start": 19993837,
+        "sequenceReference": {
+            "refgetAccession": "SQ._0wi-qoDrvram155UmcSC-zA5ZK4fpLT",
+            "type": "SequenceReference"
+        },
+        "type": "SequenceLocation"
+    },
+    "state": {
+        "length": 4,
+        "repeatSubunitLength": 2,
+        "sequence": "GTGT",
+        "type": "ReferenceLengthExpression"
+    },
+    "type": "Allele"
+}
+
+
 @pytest.mark.vcr
 def test_from_beacon(tlr):
+    tlr.normalize = False
     assert tlr._from_beacon(snv_inputs["beacon"]).model_dump(exclude_none=True) == snv_output
 
 
 @pytest.mark.vcr
 def test_from_gnomad(tlr):
+    tlr.normalize = False
     assert tlr._from_gnomad(snv_inputs["gnomad"]).model_dump(exclude_none=True) == snv_output
+    assert tlr._from_gnomad(deletion_inputs["gnomad"]).model_dump(exclude_none=True) == gnomad_deletion_output
+    assert tlr._from_gnomad(insertion_inputs["gnomad"]).model_dump(exclude_none=True) == gnomad_insertion_output
+    assert tlr._from_gnomad(duplication_inputs["gnomad"]).model_dump(exclude_none=True) == duplication_output
 
+    tlr.normalize = True
+    assert tlr._from_gnomad(snv_inputs["gnomad"]).model_dump(exclude_none=True) == snv_output
+    assert tlr._from_gnomad(deletion_inputs["gnomad"]).model_dump(exclude_none=True) == deletion_output_normalized
+    assert tlr._from_gnomad(insertion_inputs["gnomad"]).model_dump(exclude_none=True) == insertion_output
+    assert tlr._from_gnomad(duplication_inputs["gnomad"]).model_dump(exclude_none=True) == duplication_output_normalized
 
 @pytest.mark.vcr
 def test_from_hgvs(tlr):
+    tlr.normalize = False
```

Comment:
Are these setting to false just because of the setting to true in the gnomad test?

Reply:
Yes. I will probably work on #231 once I have time

Reply:
I agree with the `This highlights the need for https://github.com/ga4gh/vrs-python/issues/231`